crates/db_schema/src/lib.rs

```diff
@@ -56,3 +56,10 @@ pub enum SearchType {
   Users,
   Url,
 }
+
+#[derive(EnumString, Display, Debug, PartialEq, Serialize, Deserialize, Clone, Copy)]
+pub enum SubscribedType {
+  Subscribed,
+  NotSubscribed,
+  Pending,
+}
```

Comment:
This PR exposes Pending in a better way, by adding a string enum for the 3 possible subscribed states.

Reply:
Agree, enum is definitely a better way to handle it.